modules/op-installing-tkn-on-windows.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/tkn_cli/installing-tkn.adoc
+
+[id="installing-tkn-on-windows"]
+
+= Installing {op-title} on Windows
+
+== Binary installation
+
+. Download the latest link:https://mirror.openshift.com/pub/openshift-v4/clients/tkn/latest/tkn-windows-amd64.exe[`tkn.exe`] file.
```

Comment:
@nkakkar81 : Most probably we will ship the `tar.gz` instead of direct binary. We will confirm it shortly and update it here.